src/App.js

```diff
@@ -1,26 +1,52 @@
 import './App.css';
 import React from 'react';
-import SearchAndFilter from './components/SearchAndFilter';
 import jwt_decode from "jwt-decode";
-import 'bootstrap/dist/css/bootstrap.min.css';
 import {Dropdown, DropdownButton}  from "react-bootstrap";
-import MyRecipes from './components/MyRecipes';
-import MyMealPlan from './components/MyMealPlan';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Header from "./components/Header";
+
+const loadScript = (src) =>
```

Comment:
why are you loading additional scripts on the fly?

Reply:
removed